server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -567,7 +570,6 @@ private Server startServer(ProjectManager pm, CommandExecutor executor,
                                @Nullable SessionManager sessionManager,
                                InternalProjectInitializer projectInitializer) {
         final ServerBuilder sb = Server.builder();
-        sb.verboseResponses(true);
```

Comment:
I found out that `verboseResponses()` was always enabled with `ServerBuilder` which may not be recommended because we came to a conclusion that stack traces should not be exposed to non-administrator users.